### PR TITLE
Add libbdplus 

### PR DESCRIPTION
**Pull request by asdfugil**

### CODE REVIEW

File: build_info/libaacs-dev.control [hunk 1]
```diff
@@ -0,0 +1,19 @@
+Package: libaacs-dev
+Version: @DEB_LIBAACS_V@
+Architecture: @DEB_ARCH@
+Priority: optional
+Section: Development
+Maintainer: @DEB_MAINTAINER@
+Depends: libaacs0 (= @DEB_LIBAACS_V@)
```
Comment (TheRealKeto) [inline build_info/libaacs-dev.control:7]: Don't "lock" or restrict dependencies to a specific version.

File: build_info/libaacs-dev.control [hunk 2]
```diff
@@ -0,0 +1,19 @@
+Package: libaacs-dev
+Version: @DEB_LIBAACS_V@
+Architecture: @DEB_ARCH@
+Priority: optional
+Section: Development
+Maintainer: @DEB_MAINTAINER@
+Depends: libaacs0 (= @DEB_LIBAACS_V@) 
+Homepage: https://www.videolan.org/developers/libaacs.html
+Tag: role::shared-lib
```
Comment (TheRealKeto) [inline build_info/libaacs-dev.control:9]: This field is outdated for **modern** packages. Remove it.

File: build_info/libaacs0.control [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+Version: @DEB_LIBAACS_V@
+Architecture: @DEB_ARCH@
+Priority: optional
+Section: Libraries
+Maintainer: @DEB_MAINTAINER@
+Depends: libgcrypt20 (>= 1.8.0), libgpg-error0 (>= 1.14)
+Recommends: libbdplus0
+Enhances: libbluray2
+Homepage: https://www.videolan.org/developers/libaacs.html
+Tag: role::shared-lib
```
Comment (TheRealKeto) [inline build_info/libaacs0.control:11]: Remove this field.

File: build_info/libbdplus-dev.control [hunk 4]
```diff
@@ -0,0 +1,19 @@
+Package: libbdplus-dev
+Version: @DEB_LIBBDPLUS_V@
+Architecture: @DEB_ARCH@
+Priority: optional
+Section: Development
+Maintainer: @DEB_MAINTAINER@
+Depends: libbdplus0 (= @DEB_LIBBDPLUS_V@)
```
Comment (TheRealKeto) [inline build_info/libbdplus-dev.control:7]: Remove version restriction.

File: build_info/libbdplus0.control [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+Version: @DEB_LIBBDPLUS_V@
+Architecture: @DEB_ARCH@
+Priority: optional
+Section: Libraries
+Maintainer: @DEB_MAINTAINER@
+Depends: libgcrypt20 (>= 1.8.0), libgpg-error0 (>= 1.14)
+Recommends: libaacs0
+Enhances: libbluray2
+Homepage: https://www.videolan.org/developers/libbdplus.html
+Tag: role::shared-lib
```
Comment (TheRealKeto) [inline build_info/libbdplus0.control:11]: Remove this line.

File: libaacs.mk [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+
+ifneq ($(wildcard $(BUILD_WORK)/libaacs/.build_complete),)
+libaacs:
+	@echo "Using previously built libaacs."
+else
+libaacs: libgcrypt gnupg libaacs-setup
+	cd $(BUILD_WORK)/libaacs && ./bootstrap && ./configure -C \
+		$(DEFAULT_CONFIGURE_FLAGS) \
+	--with-libgcrypt-prefix=$(BUILD_BASE)$(MEMO_PREFIX)$(MEMO_SUB_PREFIX) \
+	--with-libgpg-error-prefix=$(BUILD_BASE)$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)
```
Comment (TheRealKeto) [inline libaacs.mk:22]: Indent these lines.

File: libbdplus.mk [hunk 7]
```diff
@@ -0,0 +1,68 @@
...
+		$(DEFAULT_CONFIGURE_FLAGS) \
+	--with-libgcrypt-prefix=$(BUILD_BASE)$(MEMO_PREFIX)$(MEMO_SUB_PREFIX) \
+	--with-gpg-error-prefix=$(BUILD_BASE)$(MEMO_PREFIX)$(MEMO_SUB_PREFIX) \
+	--without-libaacs
+	+$(MAKE) -C $(BUILD_WORK)/libbdplus
+	+$(MAKE) -C $(BUILD_WORK)/libbdplus install \
+		DESTDIR=$(BUILD_STAGE)/libbdplus
+	+$(MAKE) -C $(BUILD_WORK)/libbdplus install \
+		DESTDIR=$(BUILD_BASE)/libbdplus
+	touch $(BUILD_WORK)/libbdplus/.build_complete
```
Comment (TheRealKeto) [inline libbdplus.mk:42]: This seems convoluted and messy. We can a better implementation for this.

File: libbdplus.mk [hunk 8]
```diff
@@ -0,0 +1,68 @@
...
+libbdplus-package: libbdplus-stage
+	# libbdplus.mk Package Structure
+	rm -rf $(BUILD_DIST)/libbdplus{0,-dev}
+	mkdir -p $(BUILD_DIST)/libbdplus{0,-dev}/$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)/lib
+	
+	# libbdplus.mk Prep libbdplus0
+	cp -a $(BUILD_STAGE)/libbdplus/$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)/lib/libbdplus.0.dylib $(BUILD_DIST)/libbdplus0/$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)/lib
+	
+	# libbdplus.mk Prep libbdplus-dev
+	cp -a $(BUILD_STAGE)/libbdplus/$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)/lib/{libbdplus.{a,la,dylib},pkgconfig} $(BUILD_DIST)/libbdplus-dev/$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)/lib
```
Comment (TheRealKeto) [inline libbdplus.mk:54]: Shorten this line. It's too long.

File: libbdplus.mk [hunk 9]
```diff
@@ -0,0 +1,68 @@
...
+	
+	# libbdplus.mk Make .debs
+	$(call PACK,libbdplus0,DEB_LIBBDPLUS_V)
+	$(call PACK,libbdplus-dev,DEB_LIBBDPLUS_V)
+
+	# libbdplus.mk Build cleanup
+	rm -rf $(BUILD_DIST)/libbdplus0
+
+.PHONY: libbdplus libbdplus-package
+
```
Comment (TheRealKeto) [inline libbdplus.mk:68]: You have 2 extra lines at the end of this file, remove this one.

**Review by TheRealKeto — CHANGES_REQUESTED** [review]

**Review by marruffohomero — APPROVED** [review]

File: build_info/libbdplus-dev.control (hunk 4 above)
Comment (Diatrus) [reply build_info/libbdplus-dev.control:7]: Don’t remove the version restriction… dev packages should be attached to their specific shlib version.

File: build_info/libaacs-dev.control (hunk 1 above)
Comment (asdfugil) [reply build_info/libaacs-dev.control:7]: libdevel packages should be locked to the same runtime version

Comment (TheRealKeto) [reply build_info/libaacs-dev.control:7]: Then discard this as resolved. This should only be the case for development packages in the future.

**Pull request closed without merging**